default_organisation.bash

```diff
@@ -212,6 +213,25 @@ cmd_recrypt() {
 	git_commit "Recrypt $* for ${target_groups[*]}."
 }
 
+# Check for an extension update
+cmd_update() {
+	echo "Looking for update..."
+	tmp_file="/tmp/pass-group-$EXTENSION_NAME.bash"
+	curl -s https://raw.githubusercontent.com/erdnaxe/pass-group/master/default_organisation.bash > $tmp_file
+	diff --color $extension $tmp_file && rm $tmp_file && echo "Already up to date." && exit 0
+	read -r -p "New update found, do you really want to update the extension? [y/N]" input
```

Comment:
Drop `really`

Reply:
And you can use `yesno` here with:
```bash
yesno "New update found, do you really want to update the extension?" || rm $tmp_file && die "Update cancelled."
```

Also check that you need to remove the file when using `mktemp`, maybe it is automatic?

Reply:
yesno does not seem to exist.
"really" got dropped.
I checked, the command "mktemp" does not remove the temp file, which is not surprising since this is a program and not a bash primitive.

Reply:
yesno exists here : https://git.zx2c4.com/password-store/tree/src/password-store.sh